### PR TITLE
Remove profile dependency from implementation code generation

### PR DESCRIPTION
**Pull request by minhaj10p**
# Description
1. Populate `ImplementsProfile` section from control excel sheet 
2. Removes dependency from profile input as arguments

### CODE REVIEW

File: templates/implementation.go [hunk 1]
```diff
@@ -1,10 +1,16 @@
 package templates
 
-import "html/template"
+import (
+	"html/template"
+)
 
 //GetImplementationTemplate gets implementation template for implementation go struct file
```
Comment (anweiss) [inline templates/implementation.go]: nit

```suggestion
// GetImplementationTemplate gets implementation template for implementation go struct file
```

**Comment by anweiss** [comment]
Looks good @minhaj10p. Please rebase and we can merge.

File: impl/implementation.go [hunk 2]
```diff
@@ -27,111 +28,106 @@ const (
...
+	parameterStringIndex int
+	uuidIndex            int
+	narrativeIndex       int
+	definition           cdMap
+	hasParameterMapping  bool
 }
 
-// GenerateImplementation generates implementation from component excel sheet
-func GenerateImplementation(CSVS [][]string, p *profile.Profile, c Catalog) implementation.Implementation {
+var profileMap = map[string]string{
```
Comment (anweiss) [inline impl/implementation.go]: @minhaj10p is there a reason why we're duplicating these profile entries?

Comment (minhaj10p) [reply impl/implementation.go]: the excel sheet had different variations of the same name. this map sanitizes it for the fedramp profiles

Comment (anweiss) [reply impl/implementation.go]: Gotcha. Thanks.

**Pull request merged**